index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="portfolio" />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="header-section">
+        <div class="header-wrapper">
```

Comment:
This might sound nitpicking but I don't think this class name is appropriate because this div tag does not wrap the header section.

Reply:
Since it wraps each element of the header, I named it as "wrapper". In order to make the code easier to manage, the other four parts(Home, Works, Skills, Contact) are given class names based on header as well.